atomkraft/cli/__init__.py

```diff
@@ -84,6 +84,7 @@ def version():
 )
 def init(
     name: str = typer.Argument(..., help="Name of new directory", show_default=False),
+    template: str = typer.Option("transfer", help="Name of project template to use"),
```

Comment:
`template` argument should be optional. Default value can be `transfer`.

Reply:
`typer.Option()` does make the argument optional with "transfer" as default here.